com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.Migration.cs

```diff
@@ -136,6 +136,7 @@ class ObsoleteFrameSettings
     public partial struct FrameSettings
     {
 #pragma warning disable 618 // Type or member is obsolete
+#pragma warning disable 612
```

Comment:
What kind of warning it is? I think it is usefull to keep a reminder of what was disabled in comment

Reply:
I think it was for obsolete property (which was not covered by the other warning)